polkadot/zombienet_tests/functional/0007-dispute-freshly-finalized.zndsl

```diff
@@ -0,0 +1,35 @@
+Description: Test if disputes triggered on finalized blocks within scope always end as valid.
+Network: ./0007-dispute-freshly-finalized.toml
+Creds: config
+
+# Check authority status and peers.
+malus: reports node_roles is 4
+honest: reports node_roles is 4
+
+# Ensure parachains are registered.
+honest: parachain 2000 is registered within 30 seconds
+honest: parachain 2001 is registered within 30 seconds
+honest: parachain 2002 is registered within 30 seconds
+honest: parachain 2003 is registered within 30 seconds
+
+# Ensure parachains made progress.
+honest: parachain 2000 block height is at least 10 within 200 seconds
+honest: parachain 2001 block height is at least 10 within 200 seconds
+honest: parachain 2002 block height is at least 10 within 200 seconds
+honest: parachain 2003 block height is at least 10 within 200 seconds
+
+# Ensure that malus is already attempting to dispute
+malus: log line contains "😈 Disputing candidate with hash:" within 180 seconds
+
+# Check if disputes are initiated and concluded.
+honest: reports polkadot_parachain_candidate_disputes_total is at least 2 within 30 seconds
```

Comment:
I think even with a larger timeout this will be flaky in CI.
```suggestion
honest: reports polkadot_parachain_candidate_disputes_total is at least 2 within 100 seconds
```

Reply:
Technically we can drop it to at least 1. This malus variant should start disputes every other finalised block (while sync backing is disabled). Within 30 seconds we are pretty much certain to get at least 2 finalizations and plenty of time to propagate the statement so does not seem that flaky to me. Where do you think the flakiness would come from?